geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/StringBytesGlossary.java

```diff
@@ -175,12 +175,13 @@
   @MakeImmutable
   public static final byte[] bABSTTL = stringToBytes("ABSTTL");
 
-  // AbstractZRangeExecutor
+  // Various ZRangeExecutors
   @MakeImmutable
   public static final byte[] bWITHSCORES = stringToBytes("WITHSCORES");
-
   @MakeImmutable
   public static final byte[] bLIMIT = stringToBytes("LIMIT");
+  public static final byte bPLUS = SIMPLE_STRING_ID; // +
```

Comment:
what is the deal with the "b" prefix? At one point I thought it meant the constant was a byte array but here it is just a byte. When should the prefix be used? Also why does it exist? I have not seen this pattern before in java.

Reply:
Prior to the refactor that moved constants out of `Coder`, the approach to naming byte/byte array constants was somewhat inconsistent, with some using the "b" prefix and some not. I can't say why this naming convention was originally chosen (commit history shows that it seems to have been present since Geode was first imported to git back in 2015), but when working on https://issues.apache.org/jira/browse/GEODE-9287, I tried to enforce consistency in all the constant names. I think I was overzealous with the single-byte values though, as you're right, many of them don't use the "b" prefix and conceptually it might make more sense to only use it for byte arrays. I can go through and rename the ones are that currently using it so that they're not if you think that'd improve things.

Reply:
I can see the danger of having byte arrays that you want to be constants. In fact any array that you want to treat as a constant could be an issue because arrays can always be modified. Should all of these static final byte arrays be marked with the Immutable annotation? For constants I'm used to all upper case and underscores so the lowercase prefix is not helpful to me. I'm open to it if it actually has a specific meaning. When I first saw it I thought we were encoding type info in the symbolic name and if so I don't see a need for that.

Reply:
See https://issues.apache.org/jira/browse/GEODE-9449 and https://issues.apache.org/jira/browse/GEODE-9450